config/settings/production.py

```diff
@@ -106,6 +108,12 @@
 CELERY_RESULT_BACKEND = CELERY_BROKER_URL
 CELERY_BROKER_USE_SSL = {"ssl_cert_reqs": ssl.CERT_REQUIRED}
 
+CELERYBEAT_SCHEDULE = {
+    "every-day-generate-geo-index": {
+        "task": "adserver.tasks.daily_update_geos",
+        "schedule": crontab(minute="*/30"),
```

Comment:
Should we actually make this a daily task? Or are you thinking this during the rollout process?

Reply:
I think it should probably be daily, but wanted it faster for the rollout, yea.